src/duct/module/web.clj

```diff
@@ -32,9 +32,9 @@
     {:duct.router/cascading []}
     {}))
 
-(def ^:private logging-config
-  {:duct.middleware.web/log-requests {}
-   :duct.middleware.web/log-errors   {}
+(defn- logging-config [config]
+  {:duct.middleware.web/log-requests (or (:duct.middleware.web/log-requests config) {})
```

Comment:
I don't understand the purpose of these changes. Why not just let the configurations merge normally? Why add this exception?

Reply:
Thank you.
That is my misunderstand of configuration.
Now I rollbacked original code.
And I added test code for changing log level via config in `test/duct/module/web_test.clj`.